eightshift-boilerplate.php

```diff
@@ -38,20 +41,20 @@
  * The code that runs during plugin activation.
  */
 register_activation_hook(
-  __FILE__,
-  function() {
-    ( new \Core\Activate() )->activate();
-  }
+	__FILE__,
+	function () {
+		( new Activate() )->activate();
+	}
 );
 
 /**
  * The code that runs during plugin deactivation.
  */
 register_deactivation_hook(
-  __FILE__,
-  function() {
-    ( new \Core\Deactivate() )->deactivate();
-  }
+	__FILE__,
+	function () {
+		( new Deactivate() )->deactivate();
+	}
```

Comment:
We don't have a plugin factory for this?

Reply:
Nope, I could add it but I believe it's fine for now to keep things simple.

Reply:
Yeah, it just looks... Messy (instantiating classes directly) 🤷🏼‍♂️

Reply:
We do the same for Main / Cli.

Reply:
Yeah, I'm not too thrilled about that as well 😂